src/tests/ftest/util/ior_test_base.py

```diff
@@ -223,17 +228,25 @@ def check_subprocess_status(self, operation="write"):
                 self.mpirun.process, self.ior_cmd):
             self.fail("Exiting Test: Subprocess not running")
 
-    def run_ior(self, manager, processes, intercept=None, display_space=True):
+    def run_ior(self, manager, processes, intercept=None, display_space=True,
+                plugin_path=None):
         """Run the IOR command.
 
         Args:
             manager (str): mpi job manager command
             processes (int): number of host processes
             intercept (str): path to interception library.
+            plugin_path (str, optional): HDF5 vol connector library path.
+                This will enable dfuse (xattr) working directory which is
+                needed to run vol connector for DAOS. Default is None.
         """
         env = self.ior_cmd.get_default_env(str(manager), self.client_log)
         if intercept:
             env["LD_PRELOAD"] = intercept
+        if plugin_path:
+            env["HDF5_VOL_CONNECTOR"] = "daos"
+            env["HDF5_PLUGIN_PATH"] = "{}".format(plugin_path)
```

Comment:
You don't need to use .format here, possibly str(plugin_path) but most likely just plugin_path

Reply:
If I need to rewrite the test to be included in ior_small as suggested above I will make this change.

Reply:
Change made in latest commit